index.html

```diff
@@ -34,6 +34,35 @@
         <a href="#about"><p>About</p></a>
         <a href="#contact"><p>Contact</p></a>
       </nav>
+    </div>
+    <div id="overlay">s
+      <div id="popup">
+        <div class="popup-header">
```

Comment:
- [ ] As the previous reviewer as earlier suggested, kindly make sure that the `font-weight` of all your popup projects title matches the Figma design you chose 😄 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/189400664-924ed054-23ec-43f7-a425-da447c5215df.png)  | ![image](https://user-images.githubusercontent.com/63473584/189400807-36fc34f3-4031-4fad-84ca-1af5db7f3ae9.png)  |